### PR TITLE
Pop up menu

### PR DESCRIPTION
**Pull request by EzekielUtshudi**
### **_In this PR i implemented :_**
-On-clicks (or taps) the button to check to see project details, the popup with details about the see project appears,
-On clicks (or taps) the close (X) button, the popup disappears,
-popups for both mobile and desktop screen sizes.

### CODE REVIEW

File: js/menu.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  modalContainer.appendChild(modal);
+}
+
+function closeBtn() {
+  document.querySelector('.modalcross').addEventListener('click', () => {
+    modalContainer.classList.remove('active');
+    document.body.classList.remove('no-scroll');
+  });
+}
+
```
Comment (gemmen29) [inline js/menu.js:115]: - Kindly try to match the Figma design for the left and right spacing for the pop-up modal.

| Your design | Figma design |
| --- | --- |
| image | image |

- Kindly try to match the close button in the mobile view to match the Figma design.
- Kindly add the ` border-radius ` for the pop-up modal to match the Figma design.
- Please try to display the image for the `see source` button.
image
- Kindly try to remove the comma between the technology list in the pop-up modal.
image

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @EzekielUtshudi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/menu.js (hunk 1 above)
Comment (EzekielUtshudi) [reply js/menu.js]: Hi, thanks for the feedback, I fixed all the above points, but regarding the button view, I cannot use white or black background due to the fact that it clashes with the IMG or body color, please check my changes before I request for the last review.

**Review by gemmen29 — APPROVED** [review]
Hi @EzekielUtshudi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**